cmd/lncli/cmd_open_channel.go

```diff
@@ -104,6 +104,19 @@ var openChannelCommand = cli.Command{
 			Name:  "local_amt",
 			Usage: "the number of satoshis the wallet should commit to the channel",
 		},
+		cli.Uint64Flag{
+			Name: "base_fee_msat",
+			Usage: "the base fee in milli-satoshis that will " +
+				"be charged for each forwarded HTLC, regardless " +
+				"of payment size",
+		},
+		cli.Uint64Flag{
+			Name: "fee_rate_ppm",
+			Usage: "the fee rate that will be charged " +
+				"proportionally based on the value of each " +
+				"forwarded HTLC, the lowest possible rate is 0 " +
+				"with a granularity of 0.000001 (millionths).",
```

Comment:
I would change this to make it explicit that this will be the rate per million. If not it looks a bit weird that you have a granularity of `0.000001` in a uint64.